doc-build/md055.md

```diff
@@ -0,0 +1,20 @@
+This rule is triggered when a table is missing a border. Borders consist of the
+pipe '|' character.
+
+This table is missing a border on the both sides:
+
+```markdown
+| Heading | Heading |
+|---------|---------
+  Data    | Data    |
+```
+
+To fix this, add a pipe character at the start or end of the row.
+
+```markdown
+| Heading | Heading |
+|---------|---------|
+| Data    | Data    |
+```
+
+Rationale: Some renderers may render tables without borders incorrectly.
```

Comment:
Is this true, by the way? Do you have any examples of scenarios where this breaks parsing? (I think it IS true, but couldn't say what breaks.)

Reply:
Not many, but apparently StackOverflow has some issues [one line table](https://meta.stackexchange.com/questions/372897/incomplete-table-markdown-causes-preview-to-appear-fine-while-its-broken-when-p), [Missing column](https://meta.stackexchange.com/questions/371475/some-markdown-tables-incorrectly-omit-an-entire-column-in-the-rendered-post-but)

Reply:
Apparently StackOverflow has some issues [one line table](https://meta.stackexchange.com/questions/372897/incomplete-table-markdown-causes-preview-to-appear-fine-while-its-broken-when-p), [Missing column](https://meta.stackexchange.com/questions/371475/some-markdown-tables-incorrectly-omit-an-entire-column-in-the-rendered-post-but)
Also, IntelliJ doesn't render an inline preview when a pipe is missing.